matron/src/clocks/clock_link.c

```diff
@@ -24,7 +23,7 @@ static struct clock_link_shared_data_t {
     pthread_mutex_t lock;
 } clock_link_shared_data;
 
-static abl_link link;
+static abl_link link_instance;
```

Comment:
a better name for this could be `clock_link` for consistency with the rest of variables.